README.md

```diff
@@ -15,9 +15,12 @@ On module insertion, you can pass following parameters to the module:
     Port you want the module to listen. If the default port is in use, or you simply want to use another port, you can use this param to specify.
   - backlog (`128` by default)
 
-    Backlog amount you want the module to use. Typically, you only need to change this if you encounter too much concurrent connections warning, which will be logged in the kernel message like this, `Possible SYN flooding on port 12345`. For details about SYN flooding, you can refer to the [SYN flood wiki](https://en.wikipedia.org/wiki/SYN_flood). Changing this param allows the kernel to handle more/less connections concurrently.
+    Backlog amount you want the module to use. Typically, you only need to change this if you encounter warning of too much concurrent connections, which will be logged in the kernel message like this, `Possible SYN flooding on port 12345`.  For details about SYN flooding, you can refer to the [SYN flood wiki](https://en.wikipedia.org/wiki/SYN_flood). Changing this param allows the kernel to handle more/less connections concurrently.
+  - bench (`0` by default)
+  
+    By setting this param to `1`, you gain better cache locality during benchmarking the module. More specifically, we use [`WQ_UNBOUND`](https://www.kernel.org/doc/html/latest/core-api/workqueue.html#flags) as workqueue creation flag for the module by default, because this flag allows you to establish both long term (use telnet-like program to interact with the module) and short term (benchmarking) connection. However, this flag has a trade-off, which is cache locality, so you can set the param to `1` to disable `WQ_UNBOUND` flag if you are only intend to run the benchmarking. BE AWARE that if you use telnet-like program to interact with the module with the param set to `1`, your machine may get unstable since your connection may stalls tasks in the system workqueue (global workqueue in the kernel).
 ```
-$ sudo insmod fastecho.ko port=<port_you_want> backlog=<amount_you_want>
+$ sudo insmod fastecho.ko port=<port_you_want> backlog=<amount_you_want> bench=<either_1_or_0>
```

Comment:
You have to explain the operations and side effects when condition `bench=1`.

Reply:
The explanation of side effects are at above (paragraph at [L21](https://github.com/sysprog21/kecho/pull/1/files/661bb56edf0fb6603370f19cccc5591a619b208d#diff-04c6e90faac2675aa89e2176d2eec7d8R21)). However, there's room for improvement, will update later.